share/picongpu/tests/compileAtomicPhysics/include/picongpu/param/atomicPhysics.param

```diff
@@ -0,0 +1,101 @@
+/* Copyright 2022-2023 Brian Marre
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ * configuration file for atomicPhysics, also see speciesDefinition.param for further
+ *  necessary settings to enable atomicPhysics.
+ */
+
+#pragma once
+
+#include "picongpu/particles/atomicPhysics/ExponentialApproximationProbability.hpp"
+#include "picongpu/particles/atomicPhysics/LinearApproximationProbability.hpp"
+#include "picongpu/particles/atomicPhysics/atomicData/AtomicData.hpp"
+#include "picongpu/particles/atomicPhysics/electronDistribution/LogSpaceHistogram.hpp"
+#include "picongpu/particles/atomicPhysics/stateRepresentation/ConfigNumber.hpp"
+
+#include <cstdint>
+
+namespace picongpu::atomicPhysics
+{
+    // configuration of electron ElectronHistogram
+    //{
+    struct MaxEnergyParam
+    {
+        // eV
+        static constexpr float_X maxEnergy = float_X(1e7);
+    };
+    using ElectronHistogram = particles::atomicPhysics::electronDistribution::LogSpaceHistogram<
+        100u, // T_numberBins
+        MaxEnergyParam>;
+    //}
+
+    // atomicPhysics rate solver settings
+    //{
```

Comment:
REFACTOR: Do I understand this correctly that the curly braces are kind of creating an implicit subsection here? If so, why not use a struct and make it explicit? If not, this is likely deada code and should be removed.

Reply:
they are supposed to indicate that the comment describes several lines, start and end indicated by the braces.
I copied this style from psychocoder but would be open to change it

Reply:
followed your suggestion, and created a param struct